### DISCUSSION TITLE
Add Manticore search to "Comparison to alternatives"

### DISCUSSION THREAD

**Discussion by lospejos** [meilisearch/product#734] [Feedback & Feature Proposal]
Could you please add [ManticoreSearch](https://manticoresearch.com) in [Comparison to alternatives](https://www.meilisearch.com/docs/learn/what_is_meilisearch/comparison_to_alternatives) topic?  Thanks!

**Comment by macraig** [comment]
Hi @lospejos , thanks for submitting your proposal. We're a bit tight on bandwidth and working on other priorities at the moment, but I'll keep the discussion open to gather feedback and upvotes from other users in the future. PRs on our docs repo are also welcome!
